helpers/is-bounty-closed.ts

```diff
@@ -0,0 +1,9 @@
+import models from "db/models";
+
+export async function isBountyClosed(id: number) {
```

Comment:
No need to add this helper. The issue model has a virtual column `isClosed` which already do this verification.
https://github.com/layerx-labs/webnetwork/blob/dev/db/models/issue.model.js#L49